stories/ag/Listings/DesktopListing.js

```diff
@@ -15,7 +15,6 @@ const props = {
   },
   photos: {
     server: 'https://image.rent.com/',
-    dimensions: '280-120',
```

Comment:
Sorry, I know this is a nitpick and a bunch of stories are currently broken anyway, but would you mind adding the dimensions to the URLs in `stories/ag/Listings/baseListing.js` so the story still has the proper dimensions?